test/e2e/egress_firewall.go

```diff
@@ -532,9 +513,12 @@ spec:
 			framework.Failf("Failed to connect from pod to external container, before egress firewall is applied")
 		}
 		// external container -> nodePort svc should work
+		svc, err := f.ClientSet.CoreV1().Services(f.Namespace.Name).Get(context.TODO(), "service-for-pods", metav1.GetOptions{})
```

Comment:
nit: Use `serviceName` variable instead of `"service-for-pods"`

It is difficult to track that `createServiceForPodsWithLabel` uses that as a service name without that being defined as a constant somewhere or being passed in as a parameter.

Reply:
ack yeah good point, let me fix this

Reply:
done.